Sources/ObjCSupport/LOKSizeLayout.swift

```diff
@@ -9,6 +9,15 @@
 import CoreGraphics
 
 @objc open class LOKSizeLayout: LOKBaseLayout {
+    public let maxWidth: CGFloat?
+    public let maxHeight: CGFloat?
+    @objc public let minWidth: CGFloat
+    @objc public let minHeight: CGFloat
+    @objc public let alignment: LOKAlignment?
```

Comment:
We should be able to make this alignment property non-optional by constructing a LOKAlignment object from the Swift SizeLayout's alignment property, because in its initializer it computes a default alignment value. This way, after the object is constructed, we can actually reflect the actual non-null alignment value that the layout has.